### PR TITLE
Add support for `Untag` feature

### PR DESCRIPTION
**Pull request by DanzaSeah**
Add support for the untag feature, which allows users to untag one or more existing tags from a contact 

Created UntagCommand class,  UntagCommandParser class and respective test classes

### CODE REVIEW

**Comment by DanzaSeah** [comment]
fixes #61

File: src/main/java/seedu/address/logic/commands/UntagCommand.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Removes one or multiple tags from the person identified "
+            + "by the index number used in the last person listing.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[TAG]... (can specify multiple tags)\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + "florist photographer.";
+
+    private final Index index;
+    private final List<Tag> tagsToRemove;
```
Comment (HanB1n) [inline src/main/java/seedu/address/logic/commands/UntagCommand.java:33]: Type of tagsToRemove could follow the type in Person.java, HashSet for consistency?

**Review by HanB1n — COMMENTED** [review]
LGTM in general. Left a comment regarding consistency.

File: src/main/java/seedu/address/logic/commands/UntagCommand.java (hunk 1 above)
Comment (DanzaSeah) [reply src/main/java/seedu/address/logic/commands/UntagCommand.java]: True, ill make the change, thanks

**Review by dasha3412 — COMMENTED** [review]
Looks well structured and well done, no bugs found.

**Pull request merged**